INN/umbrella-citylimits
# Remove child-style.less, make grunt use style.less

## Changes

This pull request makes the following changes:

- Copies over content of `child-style.less` to `style.less`
- Removes `child-style.less`
- Updates `gruntfile.js` to pull from `less/style.less` instead of `less/child-style.less`.

## Why



For #59 

## Testing/Questions

Features that this PR affects:

- Sitewide CSS

Steps to test this PR:

1. Run grunt and make sure it pulls from the `style.less` file
2. Make sure no styling looks off on the frontend
